src-built-in/components/appLauncher2/src/components/Content.jsx

```diff
@@ -50,15 +64,15 @@ export default class Content extends React.Component {
 		})
 	}
 
-	onActiveFolderChanged(error, data) {
+	async onActiveFolderChanged(error, data) {
```

Comment:
This is going to be a general comment for this file. You should not need all of the asyncs. If you're using the store state should be updated from a listener.

Reply:
You already have a listern for this section `store.addListener({ field: 'activeFolder' }, this.onActiveFolderChanged)`. No need for async.

Reply:
We can't, activeFolder is a string, while content requires the active folder object. The eventListener will return the name of the folder only.